### ISSUE TITLE
00 -- Jasmine Baik

### ISSUE THREAD

**Issue by boshnivolo** [TIY-Durham/2015-FALL-FEE#22]
- [x] **GitHub-ing to Know You**
  - [x] [Create a GitHub account](https://github.com/signup)
  - [x] [Edit your GitHub profile](https://github.com/settings/profile) to provide:
  - [x] A picture, so we can recognize your face.
  - [x] Your full name, so we can recognize your name.
  - [x] A valid public email address, so we can contact you.
  - [x] Find and star [the class project](https://github.com/TIY-Durham/2015-FALL-FEE)
  - [x] Read [the Home page on the class wiki](https://github.com/TIY-Durham/2015-FALL-FEE/wiki)
- [x] **You've Got Issues**
  - [x] [Create a new _WIP Issue_](https://github.com/TIY-Durham/2015-FALL-FEE/issues/new)
  - [x] Name it `00 -- YOUR NAME` (where `YOUR NAME` is _your_ name)
  - [x] Make this checklist the description
  - [x] Start checking off boxes... :astonished:
- [x] **Send all my GitHub to you...**
  - [x] Learn all you can about `git` and GitHub!
    - Treehouse has [a Git Basics course](http://teamtreehouse.com/library/git-basics)
  - CodeSchool has [a `git` path](https://www.codeschool.com/paths/git)
    - [x] Complete: [Try Git](http://try.github.com)
  - [x] GitHub has a couple of Guides:
    - [x] [Hello World](https://guides.github.com/activities/hello-world/)
    - [x] [Forking Projects](https://guides.github.com/activities/forking/)
  - [x] Add a link to your `Hello-World` and `Fork-and-Spoon` repos as a comment on your _WIP Issue_
- [x] **Command Line Training**
  - [x] Read: http://eddywashere.com/blog/an-intro-to-the-os-x-command-line/
  - [ ] Complete: https://www.codecademy.com/courses/learn-the-command-line/
  - [ ] **BEAST MODE** http://cli.learncodethehardway.org/book/
- [x] **INSTALL ALL THE THINGS!**
  - [x] [Install Homebrew](http://brew.sh) on your Mac
  - [x] Open `Terminal.app`; Don't Panic.
  - [x] [Install Homebrew and the `brew` command](http://brew.sh/#install)
  - [x] Read [the Homebrew Wiki](https://github.com/Homebrew/homebrew/tree/master/share/doc/homebrew#readme) to learn more about Homebrew
  - [x] Using the `brew` command, install the following packages:
    - [x] `zsh`
  - [x] `git`
  - [x] `hub`
  - [x] `node`
  - [x] Add the output of the following commands as comments on your _WIP Issue_:
    - [x] `which zsh && zsh --version`
    - [x] `which git && git --version`
    - [x] `which hub && hub --version`
    - [x] `which node && node --version`
  - [x] Install the following (bonus if you use `brew`):
    - [x] [Google Chrome](http://google.com/chrome)
    - [x] [Atom](http://atom.io)
    - [x] [Dash](http://kapeli.com/dash)
    - [x] [Slack](http://www.slack.com)
    - [x] [SourceTree](https://www.sourcetreeapp.com/)
- [x] **Prezto Change-oh!**
  - [x] Find and _fork_ the Prezto project on GitHub
  - [x] Clone _your fork_ of Prezto into your home directory
  - [x] Follow the installation instructions in the `README.md` file
- [x] **Take a Moment to Reflect**
  - [x] Answered question 1 in a comment on your WIP Issue.
  - [x] Answered question 2 in a comment on your WIP Issue.

**Comment by boshnivolo** [comment]
https://github.com/boshnivolo/Spoon-Knife
https://github.com/boshnivolo/hello-world

**Comment by boshnivolo** [comment]
which zsh && zsh --version
/bin/zsh
zsh 5.0.5 (x86_64-apple-darwin14.0)

which git && git --version
/usr/bin/git
git version 2.3.2 (Apple Git-55)

which hub && hub --version
hub not found

which node && node --version                                            ⏎
/usr/local/bin/node
v0.12.7

**Comment by boshnivolo** [comment]
1. I imagine I'll have difficultly taking in all this new information so quickly, but I can work through it by immersing myself with all the available resources offered to me online and in class. A similar situation to this is when I began learning English. Though born in America, I spent my toddler years with my grandmother in Korea, so once I traveled back to America to start school, I faced a pretty big language barrier. I struggled for a few years, but by taking in as much content around me (television, books, music, etc.) my English became stronger. Now, it feels natural to me, as if it were my first language. 

Though I may not be sure if I will get the same results with this course, I will keep trying my best in order to make coding feel natural to me too.

**Comment by boshnivolo** [comment]
1. The pre-work that I found most enjoyable was probably the courses on Codecademy. Being able to see my HTML and CSS transform right in front of me helped me solidify my actions with the outcomes. They also have a really nice interface that combined functionality with flare quite nicely. 

My LEAST favorite though was probably trying to interpret the terminal's output with brew commands and zsh. I found it rather confusing at first, and am still a little unsure if I correctly downloaded everything, but I intend to read up on it in order to understand it better.
